_delphi_utils_python/delphi_utils/validator/static.py

```diff
@@ -178,17 +178,17 @@ def check_bad_geo_id_value(self, df_to_test, filename, geo_type, report):
             - report: ValidationReport; report where results are added
         """
         valid_geos = self._get_valid_geo_values(geo_type)
-        unexpected_geos = [geo for geo in df_to_test['geo_id']
-                           if geo.lower() not in valid_geos]
+        unexpected_geos = list(set([geo for geo in df_to_test['geo_id']
```

Comment:
Made the "invalid values" list unique and in a couple other places (let me know if there's a better way to do this!)